02_Util/src/util/swagger.ts

```diff
@@ -36,11 +46,13 @@ function OcppTransformObject({
             methodKey
           ] as OpenAPIV3.OperationObject;
           if (method) {
-            // Set tags based on path key
-            method.tags = pathKey
-              .split('/')
-              .slice(2, -1)
-              .map((tag) => tag.charAt(0).toUpperCase() + tag.slice(1));
+            // Set tags based on path key if tags were not passed in
+            if (!method.tags) {
```

Comment:
adjusting to instead have current logic as fallback and if tags are passed in then they are honored